cognite/client/data_classes/transformations/__init__.py

```diff
@@ -27,13 +27,14 @@
 from cognite.client.data_classes.transformations.schedules import TransformationSchedule
 from cognite.client.data_classes.transformations.schema import TransformationSchemaColumnList
 from cognite.client.exceptions import CogniteAPIError
+from cognite.client.utils._logging import COGNITE_LOGGER_NAME
 from cognite.client.utils._text import convert_all_keys_to_camel_case
 
 if TYPE_CHECKING:
     from cognite.client import CogniteClient
 
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(COGNITE_LOGGER_NAME)
```

Comment:
This one should remain as is - the warning should always be emitted.

Reply:
Aha, I see. Should the warnings module be used instead to be explicit?
https://docs.python.org/3/library/warnings.html

Reply:
Yes, I think that would be nice 👌

Reply:
Took a closer look at the code.
are you talking about this message that it should always be emitted?
```python
logger.debug(
  f"Unable to create a session and get a nonce towards {project=} using the provided "
  f"{credentials_name} credentials: {err!r}"
)
```
Not sure what you mean by "emitted" but as far as I understand it will not be shown by by default, because the default logger level is INFO. And this won't be shown even if we enable debug logging because this is not using a cognite sdk logger or a child logger 🤷

Reply:
I wrote that log message - and I regret not using a `warning` instead. I don't see how or when a user specifying to use nonce credentials would not want to know about OIDC credentials being used instead (passed to, and stored in the transformation backend).

I vote for `UserWarning` 👍 

Please also make it a bit more verbose, perhaps adding an extra sentence
`The provided OIDC credentials will be passed on to the transformation service.`

Reply:
updated to use user warning